js/app.js

```diff
@@ -48,3 +48,27 @@ const sideNavExpansion = function (event) {
 for (let i = 0; i < sideNavParents.length; i++) {
   sideNavParents[i].addEventListener('click', sideNavExpansion, false);
 }
+
+
```

Comment:
Can this be done more simply, with a parent class that handles the toggling? this is a lot of dom manipulation for what is effectively a few CSS changes.

Reply:
Probably, but if I make changes here I can't test them as the sidenav doesn't exist in this branch.  We would have to wait until the PR for branch chore-dynamic-sidenav is up for review--that branch and PR does not exist yet.  This block is a 1:1 from the Jekyll site with minimal changes to make it continue to function.  Please let me know how you'd like me to proceed.